lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -304,6 +304,7 @@
       <converter file="archive_to_directory.xml" target_datatype="directory"/>
     </datatype>
     <datatype extension="directory" type="galaxy.datatypes.data:Directory"/>
+    <datatype extension="bwa_mem2_index" display_in_upload="true" type="galaxy.datatypes.data:Directory"/>
```

Comment:
Wondering if we should subclass and have some basic sanity checks on the contents. At the moment users can upload anything, or?